### PR TITLE
Use alias + mx + txt

### PR DESCRIPTION
**Pull request by sometgirldotonline**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by novathefox** [comment]
why tf is it complaining abt cname/a records when i dont have them

File: domains/novafox.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
         "email": "foxsdenyt@gmail.com"
     },
     "record": {
-        "CNAME": "foxsdenyt-github-io.onrender.com"
+        "ALIAS": "foxsdenyt-github-io.onrender.com",
```
Comment (Stef-00012) [inline domains/novafox.json:7]: This isn't a valid record that you can use here

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/novafox.json (hunk 1 above)
Comment (novathefox) [reply domains/novafox.json:7]: ill have to swap it to an A record at "216.24.57.1", which cant be direct access and has to come from a domain, see: https://docs.render.com/configure-other-dns)

File: domains/novafox.json [hunk 2]
```diff
@@ -4,6 +4,8 @@
         "email": "foxsdenyt@gmail.com"
     },
     "record": {
-        "CNAME": "foxsdenyt-github-io.onrender.com"
+        "A":"216.24.57.1",
```
Comment (CuteDog5695) [inline domains/novafox.json:7]: IP Address needs to be in an array

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**